src/components/markdown-renderer/hooks/use-markdown-extensions.ts

```diff
@@ -103,6 +102,10 @@ export const useMarkdownExtensions = (
       new HighlightedCodeMarkdownExtension(),
       new DebuggerMarkdownExtension()
     ]
+    if (plantumlServer) {
+      extensions.push(new PlantumlMarkdownExtension(plantumlServer))
+    }
```

Comment:
Why was this behaviour changed? If no plantuml server has been configured then the extensions should render an error message. With this change plantuml will just be ignored.

Reply:
Okay, there was a type error because the extension required a string therefore I did it like this, but it seems better to adjust the extension instead.

Reply:
Done